code/API_definitions/qod-api.yaml

```diff
@@ -430,6 +444,44 @@ components:
           - address - an IPv4 number in dotted-quad form 1.2.3.4. Only this exact IP number will match the flow control rule.
           - address/mask - an IP number as above with a mask width of the form 1.2.3.4/24.
             In this case, all IP numbers from 1.2.3.0 to 1.2.3.255 will match. The bit width MUST be valid for the IP version.
+    PublicIpv4Addr:
+      description: |
+        Public (routable) source IP address and public port seen by, and identifying the device to, the application server
+      type: object
+      properties:
+        public_address:
+          $ref: "#/components/schemas/SingleIpv4Addr"
+        public_port:
+          $ref: "#/components/schemas/Port"
+      required:
+        - public_address
+        - public_port
+      example:
+        {
+          "public_address": "85.255.235.230",
+          "public_port": 60276
```

Comment:
In the case when both `public_ipv4_addr` & `private_ipv4_addr` are the same, what should be the value of `public_port`?